webapp/packages/core-ui/src/ContextMenu/MenuActionElement.tsx

```diff
@@ -70,7 +70,7 @@ export const MenuActionElement = observer<IMenuActionElementProps>(function Menu
       >
         <MenuItemElement
           label={actionInfo.label}
-          icon={<Checkbox checked={checked} mod={['primary', 'small']} ripple={false} />}
+          icon={<Checkbox variant="secondary" checked={checked} size="small" />}
```

Comment:
Our usual context menu has a white background, so we can't use "secondary" style.

<img width="533" alt="Screenshot 2025-06-03 at 11 18 57 PM" src="https://github.com/user-attachments/assets/9450adb6-b37b-46b5-9485-58b8ede23058" />
<img width="537" alt="Screenshot 2025-06-03 at 11 19 17 PM" src="https://github.com/user-attachments/assets/51903d6f-3d6c-419c-9390-b6b8d602f5ea" />